appveyor.yml

```diff
@@ -17,3 +17,7 @@ build: false
 
 test_script:
   - cmd.exe /c ci-scripts\windows-tests.cmd
+  # Pathfinder builds servo-freetype-sys as a dependency which has additional
+  # build requirements. So we do it in a separate .cmd file that we skip over
+  # in Firefox CI for now.
```

Comment:
I'll move this comment to the commit message for the patch. I already have a solution in the works that will let us run the pathfinder build in Firefox CI but it may take an extra day or two to propagate the necessary patches from upstream repos. We won't need to touch this file again when it starts working in Firefox CI so this comment will be obsolete at that point.